code/.storybook/vitest.config.ts

```diff
@@ -33,7 +33,7 @@ export default mergeConfig(
       include: [
         // TODO: test all core and addon stories later
         // './core/**/components/**/*.{story,stories}.?(c|m)[jt]s?(x)',
-        '../addons/interactions/src/**/*.{story,stories}.?(c|m)[jt]s?(x)',
+        '../addons/**/src/**/*.{story,stories}.?(c|m)[jt]s?(x)',
```

Comment:
All of them work now ?!?

Reply:
The addons, yes! There was a flake but I fixed it.
Most of the problems come from the other part (core) which is not enabled yet